packages/message-encryption/package.json

```diff
@@ -62,9 +62,9 @@
     "check:lint": "eslint src *.js",
     "check:spelling": "cspell \"{README.md,src/**/*.ts}\"",
     "check:tsc": "tsc -p tsconfig.dev.json",
-    "test": "run-s test:*",
-    "test:node": "TS_NODE_PROJECT=./tsconfig.dev.json mocha",
-    "test:browser": "karma start karma.conf.cjs",
+    "test": "NODE_ENV=test run-s test:*",
+    "test:node": "NODE_ENV=test TS_NODE_PROJECT=./tsconfig.dev.json mocha",
+    "test:browser": "NODE_ENV=test karma start karma.conf.cjs",
```

Comment:
is it confirm this env variable is used with karma?

Reply:
shouldn't be a problem against karma specifically since this flag is used by the SDK -- we are okay even not disabling info log for karma tests for now